stock_quant_task_deferred/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,2 @@
+- Denis Roussel <denis.roussel@acsone.eu>
+- Jacques-Étienne Baudoux <je@bcim.be>
```

Comment:
Not sure if I should be listed as well? As I gave the idea to J-E (I implemented that for one of our customer).